ansible/modules/hashivault/hashivault_write.py

```diff
@@ -85,20 +85,22 @@ def main():
 
 
 @hashiwrapper
-def hashivault_write(params):
+def hashivault_write(module):
     result = {"changed": False, "rc": 0}
-    client = hashivault_auth_client(params)
-    secret = params.get('secret')
+    client = hashivault_auth_client(module.params)
+    secret = module.params.get('secret')
```

Comment:
Not a huge deal, but seems like it woud be easier to create a local `params = module.params` here fewer changes after that